docs/orchestrator.md

```diff
@@ -215,61 +215,7 @@ code: 0
 codespace: ""
 data: 12300A2E2F63656C65737469612E7167622E76312E4D7367526567697374657245564D41646472657373526573706F6E7365
 events:
-- attributes:
```

Comment:
[note to reviewers]
I think this is just noise in the documentation that requires readers to scroll unnecessarily to continue reading. It doesn't have any important information, so I deleted it

Reply:
agreed